lib/artworkselection.py

```diff
@@ -78,7 +79,7 @@ def onInit(self):
             self.guilist = self.getControl(6)
             for arttype in self.arttypes:
                 listitem = xbmcgui.ListItem(arttype['label'])
-                summary = '{0} available'.format(arttype['count'])
+                summary = '{0} '.format(arttype['count'])+L(AVAILABLE).format(self.medialabel)
```

Comment:
Please change the language string to the whole `{0} available` string; it might need to be rearranged for another language. The second `format` also shouldn't be there. Finally, add the second PR to this one, in the same commit.